solutions/find_min_max.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for generating the min and max of a list of numbers.
+
+Module contents:
+    > find_min_max: generates the min and max of a list of numbers.
```

Comment:
Maybe changing the word generates to finding will be easier to understand.
```suggestion
    > find_min_max: generates the min and max of a list of numbers.
```

Reply:
You're right! It maintains consistency with the function name.

Reply:
Fixed this!